src/logging.ts

```diff
@@ -19,7 +19,7 @@ if (APP_VERSION.indexOf('AIV') > -1 &&
 
 function log (this: PureCloudWebrtcSdk, level, message, details?: any) {
   // immediately log it locally
-  this.logger[level](message, details);
+  this.logger[level]('SDK:', message, details);
```

Comment:
might be worth just making this `webrtc-sdk` or something so it's easier to filter in sumo logs.